securitycenter/snippets_v2/mute_findings_v2.py

```diff
@@ -76,30 +76,33 @@ def set_unmute_finding(finding_path: str) -> None:
 
 
 # [START securitycenter_bulk_mute_v2]
-# def bulk_mute_findings(parent_path: str, location_id, mute_rule: str) -> None:
-#     """
-#       Kicks off a long-running operation (LRO) to bulk mute findings for a parent based on a filter.
-#       The parent can be either an organization, folder, or project. The findings
-#       matched by the filter will be muted after the LRO is done.
-#     Args:
-#         parent_path: use any one of the following options:
-#                      - organizations/{organization}
-#                      - folders/{folder}
-#                      - projects/{project}
-#         mute_rule: Expression that identifies findings that should be updated.
-#     """
-#     from google.cloud import securitycenter_v2
-
-#     client = securitycenter_v2.SecurityCenterClient()
-
-#     request = securitycenter_v2.BulkMuteFindingsRequest()
-#     request.parent = parent_path + "/locations/" + location_id
-#     # To create mute rules, see:
-#     # https://cloud.google.com/security-command-center/docs/how-to-mute-findings#create_mute_rules
-#     request.filter = mute_rule
-
-#     response = client.bulk_mute_findings(request)
-#     print(f"Bulk mute findings completed successfully! : {response}")
-#     return response
+
+
+def bulk_mute_findings(parent_path: str, location_id, mute_rule: str) -> None:
+    """
+      Kicks off a long-running operation (LRO) to bulk mute findings for a parent based on a filter.
+      The parent can be either an organization, folder, or project. The findings
+      matched by the filter will be muted after the LRO is done.
+    Args:
+        parent_path: use any one of the following options:
+                     - organizations/{organization}
+                     - folders/{folder}
+                     - projects/{project}
+        mute_rule: Expression that identifies findings that should be updated.
```

Comment:
```suggestion
    Kicks off a long-running operation (LRO) to bulk mute findings for a parent based on a filter.

    The parent can be either an organization, folder, or project. The findings
    matched by the filter will be muted after the LRO is done.

    Args:
        parent_path: use any one of the following options:
                     - organizations/{organization}
                     - folders/{folder}
                     - projects/{project}
        mute_rule: Expression that identifies findings that should be updated.
```

Reply:
Addressed.

Reply:
See https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings - the description section (first section before Args) doesn't need to be indented.